packages/babel-preset-env/src/normalize-options.js

```diff
@@ -267,12 +267,11 @@ export default function normalizeOptions(opts: Options) {
 
   checkDuplicateIncludeExcludes(include, exclude);
 
-  const shippedProposals =
-    validateBoolOption(
-      TopLevelOptions.shippedProposals,
-      opts.shippedProposals,
-      false,
-    ) || corejs.proposals;
+  const shippedProposals = validateBoolOption(
+    TopLevelOptions.shippedProposals,
+    opts.shippedProposals,
+    false,
+  );
```

Comment:
Isn't this a breaking change?

Reply:
Practically it is not a breaking change, because we have never added any plugins to `shippedProposals`. So even if `corejs.proposals` implies `shippedProposals`, our user can not observe the effect. I would consider this change as bugfix.